2-ui/3-event-details/1-mouse-events-basics/01-selectable-list/solution.view/index.html

```diff
@@ -1,62 +1,58 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>
 <html>
-
```

Comment:
You shouldn't delete new lines since they'll cause conflicts. (line numbers should be the same.)